checkin.js

```diff
@@ -11,34 +11,12 @@
 const AWS = require('aws-sdk');
 const async = require('async');
 const m_alServiceC = require('al-collector-js/al_servicec');
+const m_alAzCollectC = require('al-collector-js/azcollectc');
```

Comment:
Looks like the two vars above aren't used anywhere.

Reply:
not sure why the linter didnt catch that., but youre right.